database/redis/src/main/java/com/example/javaopencommerce/order/CardRepositoryImpl.java

```diff
@@ -5,11 +5,13 @@
 import com.example.javaopencommerce.catalog.dtos.ItemDto;
 import lombok.extern.log4j.Log4j2;
 
+import javax.enterprise.context.ApplicationScoped;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 @Log4j2
+@ApplicationScoped
```

Comment:
Not a fan of that annotation. It would be better to create bean with a @Producer.